client/src/templates/Challenges/redux/execute-challenge-saga.js

```diff
@@ -153,11 +153,11 @@ function* executeTests(testRunner, tests, testTimeout = 5000) {
         newTest.err = 'Test timed out';
         newTest.message = `${newTest.message} (${newTest.err})`;
       } else {
-        const { message, stack } = err;
+        const { message, stack } = err?.err ?? err;
```

Comment:
```suggestion
        let { message, stack } = err?.err ?? err;
        if (message.startsWith('<!DOCTYPE')) {
          message = '';
        }
```
@ShaunSHamilton - this is a quick fix for handling a 404 page being sent back to us as an error.